daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/PhaseOne.scala

```diff
@@ -13,12 +13,11 @@ import com.daml.lf.language.{LookupError, PackageInterface}
 import com.daml.lf.speedy.Compiler.{ProfilingMode, StackTraceMode, CompilationError}
 import com.daml.lf.speedy.SBuiltin._
 import com.daml.lf.speedy.SValue._
+import com.daml.lf.speedy.SExpr0._
```

Comment:
In the case of `Phase1`, we are transforming from `Expr` to `SExpr`, and so `s` which stood for _source_ is just wrong. And no prefix is needed anyway in this case because the constructors for `Expr` and `SExpr` do not clash.

I realise that dropping the `s.` was not directly related to this work, and considered pulling the change out as a separate PR, but judged that in this case it was not worth the bother since many of the occurrences were contained in places which were being completely rewritten anyway. But happy to split this out if you want.

Reply:
makes sense, thanks for the explanation